lib/matplotlib/lines.py

```diff
@@ -1302,97 +1312,99 @@ def update_from(self, other):
         self._marker = MarkerStyle(marker=other._marker)
         self._drawstyle = other._drawstyle
 
+    @docstring.interpd
     def set_dash_joinstyle(self, s):
         """
-        Set the join style for dashed lines.
+        How to join segments of the line if it `~Line2D.is_dashed`.
 
         Parameters
         ----------
-        s : {'miter', 'round', 'bevel'}
-            For examples see :doc:`/gallery/lines_bars_and_markers/joinstyle`.
+        s : `.JoinStyle`
+            Testing
```

Comment:
What is this?

Reply:
An out-of-hand vim macro, thanks for the catch!